Pipfile

```diff
@@ -4,9 +4,4 @@ pytest = "*"
 sphinx = "*"
 
 [packages]
-humanize = "*"
-pytz = "*"
-dateparser = ">=0.7.0"
-tzlocal = "*"
-pendulum = ">=1.0"
-snaptime = "*"
+"e1839a8" = {path = ".", editable = true}
```

Comment:
That's because it doesn't work. The hash used here was just an example in https://github.com/pypa/pipenv/issues/1263#issuecomment-368200522